### PR TITLE
Create 'About us' page

### PR DESCRIPTION
**Pull request by igorstalmach**
- created 'About us' subpage
- managed to reuse components from previous work
- responsive view

### CODE REVIEW

File: src/pages/about-us/assets/about-us.module.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+@import '../../../../src/scss/breakpoints.module';
+@import '../../../../src/scss/fonts.module';
+
+body {
```
Comment (JanZmyslowski) [inline src/pages/about-us/assets/about-us.module.scss:4]: I am not sure if applying styles to the body in that file is okay

**Review by JanZmyslowski — CHANGES_REQUESTED** [review]

File: src/pages/about-us/AboutUs.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                <div className={styles.secondIconCircle}>
+                    <IconCircle/>
+                </div>
+            </div>
+            <div className={styles.wrapper}>
+                <div className={styles.header}>
+                    {__('aboutUsPage.header')}
+                </div>
+                <div className={styles.divider}></div>
+                <div className={styles.paragraph}>
```
Comment (JanZmyslowski) [inline src/pages/about-us/AboutUs.tsx:27]: Render paragraphs from array, you can add some boolean to indicate if render image on left/right side

**Review by JanZmyslowski — CHANGES_REQUESTED** [review]

File: src/pages/about-us/assets/about-us.module.scss (hunk 1 above)
Comment (igorstalmach) [reply src/pages/about-us/assets/about-us.module.scss]: true, i moved it to App.tsx so it's applied globally

**Review by JanZmyslowski — APPROVED** [review]

**Comment by JanZmyslowski** [comment]
Conflicts

**Pull request merged**